src/gui/screen_bed_leveling.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "gui.hpp"
+#include "status_footer.hpp"
+#include "window_text.hpp"
+#include "window_icon.hpp"
+#include "window_term.hpp"
+#include "window_progress.hpp"
+#include "screen.hpp"
+#include "ScreenHandler.hpp"
+
+class screen_bed_leveling_t : public AddSuperWindow<screen_t> {
```

Comment:
Use CamelCase for the name of the screen.

Reply:
Renamed